src/ng/filter/filter.js

```diff
@@ -196,7 +196,7 @@ function deepCompare(actual, expected, comparator, matchAgainstAnyProp) {
       } else if (expectedType === 'object') {
         for (key in expected) {
           var expectedVal = expected[key];
-          if (isFunction(expectedVal)) {
+          if (isFunction(expectedVal) || isUndefined(expectedVal)) {
```

Comment:
This broke things in our project. It changed the behaviour of `$filter`. I don't think it is OK to change the behaviour in minor releases.